<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImpFileSelectPage</class>
 <widget class="QWizardPage" name="ImpFileSelectPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Где находится файл дампа:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="locationBox">
         <item>
          <property name="text">
           <string>На моей машине</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>В каталоге expdp того же сервера, на который заливаем</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Уже находится в каталоге impdp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>На другом сервере БД</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="infoEdit">
         <property name="text">
          <string>Каталог на моей машине:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="pathEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="expSelectButton">
           <property name="toolTip">
            <string>Выбор файла</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../FmtLib/res/fmt.qrc">
             <normaloff>:/img/openfolderHS.png</normaloff>:/img/openfolderHS.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="dumpLabel">
           <property name="text">
            <string>Файл дампа:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="dumpEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="fileButton">
             <property name="toolTip">
              <string>Выбор файла</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../FmtLib/res/fmt.qrc">
               <normaloff>:/img/VCProject.dll_I000d_0409.ico</normaloff>:/img/VCProject.dll_I000d_0409.ico</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Владелец схемы:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ownerEdit"/>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="importLabel">
         <property name="text">
          <string>Каталог импорта:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="importEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>135</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../FmtLib/res/fmt.qrc"/>
 </resources>
 <connections/>
</ui>
